lib/archethic/db/embedded_impl.ex

```diff
@@ -200,6 +200,19 @@ defmodule Archethic.DB.EmbeddedImpl do
     ChainReader.get_transaction_chain(address, fields, opts, db_path())
   end
 
+  @doc """
+  Get a transaction chain - anti-chronological order
+
+  The returned values will be splitted according to the pagination state presents in the options
+  """
+  @spec get_transaction_chain_desc(address :: binary(), fields :: list(), opts :: list()) ::
+          {transactions_by_page :: list(Transaction.t()), more? :: boolean(),
+           paging_state :: nil | binary()}
+  def get_transaction_chain_desc(address, fields \\ [], opts \\ [])
+      when is_binary(address) and is_list(fields) and is_list(opts) do
```

Comment:
In erlang, when chaining logical test with `and`, it used to be better to use `and_also` as the tests evaluation will then stop as soon as one of the subtest fails.
I think it is still the case : https://elixirforum.com/t/why-not-and-andalso-and-or-orelse/22156

Reply:
There is no andalso in elixir